### PR TITLE
Update windows_faq.rst

### PR DESCRIPTION
**Pull request by sirredbeard**
Update regarding WSL.

Windows Subsystem for Linux is now officially supported by Microsoft. It is out of beta and will be shipping on Windows Sever 2019.

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

https://docs.microsoft.com/en-us/windows/wsl/install-win10
https://docs.microsoft.com/en-us/windows/wsl/enterprise

+label: docsite_pr

### CODE REVIEW

File: docs/docsite/rst/user_guide/windows_faq.rst [hunk 1]
```diff
@@ -36,8 +36,7 @@ Can Ansible run on Windows?
 No, Ansible cannot run on a Windows host and can only manage Windows hosts, but
 Ansible can be run under the Windows Subsystem for Linux (WSL).
 
-.. note:: The Windows Subsystem for Linux is not supported by Microsoft or
-    Ansible and should not be used for production systems. 
+.. note:: The Windows Subsystem for Linux is not supported by Ansible.
```
Comment (jborean93) [inline docs/docsite/rst/user_guide/windows_faq.rst]: I would keep the `and should not be used for production systems.`. It's ok for people to use it as a dev environment and it works perfectly well we just don't recommend/support it in an actual production like setting.

Comment (sirredbeard) [reply docs/docsite/rst/user_guide/windows_faq.rst]: I understand. Thank you.

**Comment by dagwieers** [comment]
I made the necessary changes to this branch.

@jborean93 Can you re-review ?

**Review by jborean93 — APPROVED** [review]
LGTM

**Comment by dagwieers** [comment]
Thanks everyone !

**Pull request merged**